import/index_java/src/org/vufind/index/GeoTools.java

```diff
@@ -79,11 +87,10 @@
                         // Note - storage in Solr follows the WENS order, but display is WSEN order
                         String result = String.format("ENVELOPE(%s,%s,%s,%s)", new Object[] { west, east, north, south });
                         geo_coordinates.add(result);
-                    }  else {
-                        logger.error(".......... Not indexing INVALID coordinates: [ {" + coords.get('d') + "} {" + coords.get('e') + "} {" + coords.get('f') + "} {" + coords.get('g') + "} ]");
-                    }
+                    } 
                 } else {
-                    logger.error(".......... Not indexing INVALID coordinates: [ {" + coords.get('d') + "} {" + coords.get('e') + "} {" + coords.get('f') + "} {" + coords.get('g') + "} ]");
+                    String msgError = "Poorly formed coordinate(s).";
```

Comment:
Do we need to define a string here, or could we just pass the string directly to the logging function in this case?

Reply:
Good point. Since the validateCoordinateValues logs the error, we can just send this message to the logging function here.